infra/charts/feast/charts/feast-jc/README.md

```diff
@@ -0,0 +1,71 @@
+feast-jc
```

Comment:
So functionally what we have created here is not much different[ to a controller in the Kubernetes](https://kubernetes.io/docs/concepts/architecture/controller/) sense. Can I propose that we rename this service from `jc` and `job coordinator` to simply `controller`?

So the scope of responsibility is that it is a localized control loop that synchronizes actual state with desired state, where currently its limited to jobs.

Reply:
renamed to `jobcontroller`